migration/003_add_deleted_at_column.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE "user" ADD COLUMN deleted_at Timestamp Without Time Zone;
```

Comment:
Could we explicily set up `DEFAULT NULL` ?

Reply:
Of course we can. But for what purpose?

Reply:
From my perspective, this approach will make code more cohesive. You are using `AND deleted_at IS NULL` clause in **Go** code, aren't you?

Reply:
I would add it as @delveper has mentioned. 
@YuriiYakovenko you are right that if we not pass its value a NULL will be stored.
However, if we add an explicit DEFAULT value (NULL in our case) this is more readable cos you don't need to know some implementation details - you just have to read the sql script.

Another point is related to the cases when data migration may happen. 
For example, you table already has records.
So, by explicitly using DEFAULT keyword you can specify what should happen with this column values in those records.

After all, if you know that there should be a default value - it's better to specify it.

Reply:
Thank you. Made appropriate edits